### ISSUE TITLE
SyntaxError: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by joseph1125** [danoc/react-use-clipboard#31]
When working with nextjs, it is founded that the server returns error as this syntax error pops out

```
import{useState as t,useEffect as o}from"react";import r from"copy-to-clipboard";export default function(c,e){const[i,n]=t(!1),u=e&&e.successDuration;return o(()=>{if(i&&u){const t=setTimeout(()=>{n(!1)},u);return()=>{clearTimeout(t)}}},[i,u]),[i,()=>{const t=r(c);n(t)}]}
^^^^^^

SyntaxError: Cannot use import statement outside a module
```

**Comment by danoc** [comment]
Thanks for the issue, @joseph1125.

I think I know what the problem is. I'll take a pass at fixing it tomorrow. In the meantime, try version v1.0.2. I don't think it'll have the same problem.

**Comment by joseph1125** [comment]
Thanks for the quick response, confirmed that v1.0.2 doesnt have this issue

**Comment by danoc** [comment]
Hi! Just released 1.0.7 that should work.

The issue was caused by the `exports` field in the `package.json` which is used for modern JavaScript. I'm not sure why it wasn't working, but I made #35 to track that.

**Comment by joseph1125** [comment]
Confirmed it was working at 1.06, will upgrade to 1.07 ASAP, many thanks
